faros_event.sh

```diff
@@ -107,20 +108,22 @@ function help() {
     echo "--deploy_status         | Yes | $deploy_statuses"
     echo "--artifact              | *2  | $artifact_uri_form"
     echo "--commit                | *2  | $commit_uri_form"
+    echo "--pull_request_number   |     | *3 | e.g. 123 (should be a number)"
```

Comment:
I think we want `*3` to be in `Req` column

Reply:
But it is never required, so maybe `--pull_request_number   |     | *3 e.g. 123 (should be a number)"

Reply:
+1 @willmarks  it's  conditionally required, that's whye there is the `*`